pkg/cmd/container/run_mount.go

```diff
@@ -258,7 +258,7 @@ func generateMountOpts(ctx context.Context, client *containerd.Client, ensuredIm
 
 		log.G(ctx).Debugf("creating anonymous volume %q, for \"VOLUME %s\"",
 			anonVolName, imgVolRaw)
-		anonVol, err := volStore.Create(anonVolName, []string{})
+		anonVol, err := volStore.CreateWithoutLock(anonVolName, []string{})
```

Comment:
The Create method now provides locking.
CreateWithoutLock emphasizes that this method MUST be guardrailed inside a locked section.